libs/coin-modules/coin-tezos/src/bridge/prepareTransaction.ts

```diff
@@ -29,7 +34,18 @@ export const prepareTransaction: AccountBridge<
   }
 
   const estimation = await getEstimatedFees({ account, transaction });
```

Comment:
Shouldn't we replace all values if we see a taquito error here instead ?

Reply:
In fact, I've had a look at our code.
if the taquito returns an error. (e.g. 500 internal error)

we catches the exception here
https://github.com/LedgerHQ/ledger-live/blob/develop/libs/coin-modules/coin-tezos/src/logic/estimateFees.ts#L134

and returns the initial value here
https://github.com/LedgerHQ/ledger-live/blob/develop/libs/coin-modules/coin-tezos/src/logic/estimateFees.ts#L147

the initial value is 0 gaslitmit and 0 estimatedFees

With the "maxBigNumber", we can set the "gasLimit", "estimatedFees"... to the value from coin config.